Automattic/wp-calypso
# Add new "Goals" step skeleton with `signup/goals-step` feature flag

The proposed changes add the new Goals capture step to the Site Setup flow.

There's no navigation logic set up yet. As a result, the new step is accessible with on the following URL with the use of a feature flag `signup/goals-step`:

`http://calypso.localhost:3000/setup/goals?siteSlug=[siteaddress]&flags=signup/goals-step`

| Desktop  | Mobile |
| ------------- | ------------- |
|  ![Markup on 2022-05-25 at 13:03:23](https://user-images.githubusercontent.com/25105483/170248130-724ea538-3252-4f80-9c20-01695158b2ae.png) | ![Markup on 2022-05-25 at 13:02:30](https://user-images.githubusercontent.com/25105483/170248165-bbc31621-823f-4437-89f5-55af43f5eb22.png) |

Project thread: pdDOJh-lY-p2

#### Changes proposed in this Pull Request

* adds `signup/goals-step` config feature flag
* adds new Goals capture Site Setup flow step

#### Testing instructions

1. Navigate to the Goals step URL
2. Review the step page on both mobile and desktop; compare with the design in Figma
3. There should be no effect to the existing Site Setup flow

Related to https://github.com/Automattic/wp-calypso/issues/63910 and https://github.com/Automattic/wp-calypso/issues/63922.
